relay-server/src/actors/global_config.rs

```diff
@@ -0,0 +1,101 @@
+use std::sync::Arc;
+use std::time::Duration;
+
+use relay_dynamic_config::GlobalConfig;
+use relay_system::{Addr, AsyncResponse, FromMessage, Interface, Sender, Service};
+
+use crate::actors::processor::EnvelopeProcessor;
+use crate::actors::project_upstream::GetProjectStates;
+use crate::actors::upstream::{SendQuery, UpstreamRelay};
+
+/// Service implementing the [`GlobalConfigResponse`] interface.
+///
+/// The service is responsible for fetching the global config and
+/// forwarding it to the services that require it.
+#[derive(Debug)]
+pub struct GlobalConfigService {
+    global_config: Arc<GlobalConfig>,
+    project_cache: Addr<EnvelopeProcessor>,
+    upstream: Addr<UpstreamRelay>,
+}
+
+/// Service interface for the [`GetGlobalConfig`] message.
+pub struct GlobalConfigResponse(Sender<Arc<GlobalConfig>>);
```

Comment:
According to conventions this would just be named `GlobalConfig` I believe, like how the projectcache interface is just named `ProjectCache`, but `GlobalConfig` is already the name of the global config itself

Reply:
If the issue is the name collision, you can name this `GlobalConfig` and use the qualified name for the other one (e.g. `relay_dynamic_config::GlobalConfig`). I don't have strong opinions, so your call on what you like most 😄

Reply:
hmm if that's the alternative then I think i'll keep it like it is now 🤔

Reply:
Let's follow the pattern we have in other services and give the interface the proper name. Right now it sounds like this is a message name and not the interface name. 

Maybe something like following would do:

```suggestion
pub struct GlobalConfiguration(Sender<Arc<GlobalConfig>>);
```

And to keep it consistent, you could use the full `GlobalConfiguration` prefix through out the entire file (e.g. `GlobalConfigurationService`  and so on)

Reply:
good idea 👍